compiler/examples/nba/force-recompute.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
```

Comment:
does this file have to be here? it's just one command.

In general, I want to keep the example folder as clean as possible (4 JS files + 2 scripts)

Reply:
DONE/good call.